packages/financial-templates-lib/src/clients/OptimisticOracleClient.ts

```diff
@@ -51,10 +84,11 @@ export class OptimisticOracleClient {
     public readonly web3: Web3,
     oracleAddress: string,
     votingAddress: string,
-    public readonly lookback = 604800 // 1 Week
+    public readonly lookback: number = 604800, // 1 Week
+    public readonly oracleType: OptimisticOracleType = "OptimisticOracle"
```

Comment:
you should do some validation on this type. check it is either `OptimisticOracle` or `SkinnyOptimisticOracle` or error

Reply:
Isn't this automatically validated by making `oracleType` have the `OptimisticOracleType` type? I'm not very familiar with typescript though so i could be very wrong